homework/index.js

```diff
@@ -0,0 +1,24 @@
+const inputValue = document.querySelector('#search');
+const searchButton = document.querySelector('.searchButton');
+const reposContainer = document.querySelector('.main__profile-repos');
+// const clientId = '1ebd35cdc9277ed1ee8c';
+// const clientSecret = '0ecdc67b56e4ca7c3b4710b60f135d52a8f11a32';
+const fetchRepo = async () => {
+  // const apiCall = await fetch(`https://api.github.com/users/${user}?clientId=${clientId}&clientSecret=${clientSecret}`);
+  const apiCall = await fetch('https://api.github.com/HackYourFuture/repos?callback=?');
+
+  const data = await apiCall.json();
+  return {
+    data,
+  };
+};
+
+const showData = () => {
+  fetchRepo(inputValue.value).then((res) => {
```

Comment:
Here you're passing something (`inputValue.value`) to the `fetchRepo` function, yet that function doesn't expect an argument. So you can remove that.

Nit-pick: when you remove the wrapper object around your data, I'd rename the parameter from `res` to `data` (or better still, `repositories`), that would make it clearer!